src/HDF5.jl

```diff
@@ -323,6 +323,13 @@ cset(::Type{ASCIIChar}) = H5T_CSET_ASCII
 
 hdf5_type_id(::Type{C}) where {C<:CharType} = H5T_C_S1
 
+# global configuration for complex support
+complex_support = true
+complex_field_names = ("r", "i")
```

Comment:
Would `("re", "im")` be better here, since that's what Julia uses?

Reply:
I would stick with the default convention h5py uses.

Reply:
@tknopp  why do you think it is  better to stick to `h5py`  convention?

Reply:
The goal of HDF5 is to be language agnostic. It should not matter what programming language you use to access a file. Unfortunately, HDF5 itself defines no standard for complex numbers. But there is a very popular package (h5py) that proposes or uses a convention. Why should this convention be different for Julia, when we can stick to the convention h5py uses?

Reply:
Sounds reasonable.  MATLAB does not have a convention. So we can use the `h5py` precedent.

We should document  this however.

Reply:
Yes the idea was to seamlessly compatible with h5py by default. If people are happy with this approach and want to merge I can add some documentation explaining the choice.

Reply:
I'm in favor of  sticking to the `h5py` precedent